exchangecommon/node_management_status.go

```diff
@@ -64,6 +68,9 @@ const (
 	STATUS_SUCCESSFUL      = "successful"
 	STATUS_FAILED_JOB      = "failed"
 	STATUS_INITIATED       = "initiated"
+	STATUS_ROLLBACK_STARTED = "rollback started"
+	STATUS_ROLLBACK_FAILED  =  "rollback failed"
+	STATUS_ROLLBACK_SUCCESSFUL = "rollback successful"
 )
```

Comment:
Please change STATUS_DOWNLOAD_FAILED from "failed download" to "download failed".